testsuite/MDAnalysisTests/analysis/test_dssp.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import glob
+import MDAnalysis as mda
+
+from MDAnalysis.analysis.dssp import DSSP
+from MDAnalysisTests.datafiles import DSSP_FOLDER
+from MDAnalysisTests.datafiles import XTC_multi_frame
```

Comment:
For analyzing a trajectory, choose `TPR` and `XTC`, and select "protein". Then test the secondary structure for first and last frame and the average.

Reply:
Added